### PR TITLE
added list of KIPs into README.md

### PR DESCRIPTION
**Pull request by d3mage**
Added list of KIPs into README.md
solves https://github.com/kadena-io/KIPs/issues/47

### CODE REVIEW

**Comment by rsoeldner** [comment]
@d3mage thank you for the PR. Could you also please link to the corresponding file?

**Comment by d3mage** [comment]
@rsoeldner Yes. I have just found that KIP-0015.md has some ambiguity. 
image
Which one is it actually, 13 or 15? I can see a PR for another KIP-015

**Comment by LindaOrtega** [comment]
@d3mage it's supposed to be KIP-15 not 13 (see the KIP-15 PR [here](https://github.com/kadena-io/KIPs/pull/29/files#diff-f943f006e9566ad59bc15daa10cc2f855e2c95b89667ca8cdacab14137795aa0R2) for more details).

If you preview that KIP it shows `13`, but the [raw](https://raw.githubusercontent.com/kadena-io/KIPs/master/kip-0015.md) markdown shows `15`.

I **_think_** there's something off with the markdown renderer with zero-delimitated-multi-digit numbers in a table, but I'm not sure.

See how [KIP-11](https://raw.githubusercontent.com/kadena-io/KIPs/master/kip-0011/kip-0011.md) does it and [KIP-17](https://raw.githubusercontent.com/kadena-io/KIPs/master/kip-0017.md):
- `KIP: 17`
- `KIP: "0011"`

Versus KIP-15 does it like:
- `KIP: 0015`

**Comment by d3mage** [comment]
@LindaOrtega @rsoeldner Can you review and approve this? I am working with this repository quite a lot and it would be convenient to have it.

**Review by rsoeldner — APPROVED** [review]

**Comment by ash-vd** [comment]
Is there a reason we're using 3 digits in the list (001) and 4 in the actual KIP (0001)?

Also I would rather see full markdown instead of the h3.

File: README.md [hunk 1]
```diff
@@ -8,3 +8,16 @@ Kadena Improvement Proposals (KIPs) are used to support development and
 crowdsource important features requests for different technologies developed by
 Kadena, namely, Chainweb, our proof-of-work blockchain, and the smart contract language
 [Pact](https://github.com/kadena-io/pact).
+
+
+<h3>List of final KIPs:</h3>
```
Comment (ash-vd) [inline README.md:13]:
```suggestion
## List of final KIPs:
```

Comment (d3mage) [reply README.md:13]: @ash-vd I think I've used 3 digits because I'm simply inattentive.

**Review by ash-vd — APPROVED** [review]

**Review by LindaOrtega — APPROVED** [review]

**Comment by LindaOrtega** [comment]
Fixes #47

**Pull request merged**